.github/workflows/golangci-lint.yml

```
@@ -17,7 +17,7 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with:
           # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
-          version: v1.33
+          version: v1.57.1
```

I'd like to use the `latest`  instead.

```suggestion
          version: "latest"
```